expected/12.16/extra/bool.out

```diff
@@ -249,8 +254,13 @@ INSERT INTO "type_BOOLEAN"(i, b) VALUES (27, 3.14159265358979);
 ALTER FOREIGN TABLE "type_BOOLEAN" ALTER COLUMN "b" TYPE bool;
 --Testcase 49: ERR - invalid float for bool column
 SELECT * FROM "type_BOOLEAN+";
-ERROR:  SQLite data affinity "real" disallowed for PostgreSQL data type "boolean"
-HINT:  In column "b" expected SQLite affinity "integer", incorrect value = '3.14159265358979'
+ERROR:  SQLite value is not compatible with PostgreSQL column data type
+CONTEXT:  foreign table "type_BOOLEAN+"
```

Comment:
This message seems too detail, how do you think about:
```
ERROR:  SQLite value is not compatible with PostgreSQL column data type
CONTEXT:  column "b" of foreign table "type_BOOLEAN+"
HINT:  In column "b" expected SQLite affinity "integer", incorrect value = '3.14159265358979'
```

Reply:
Yes, @nxhai98, not all of this is `CONTEXT`. `errhint` also can work in this function. I think any PostgreSQL data context elements we can leave in `CONTEXT`, but SQLite value details can move to `HINT`, are you agree?

Reply:
Refactored in https://github.com/pgspider/sqlite_fdw/pull/91/commits/6a70cc999e37b76b2eb93ea030f5b11ab587c10e based on `CONTEXT` is more detailed and about stable PostgreSQL environment, but `HINT` is a message with unstable format about SQLite value, which can printable or not, quotable or not, original or hexadeciamal, with or without useful data length.

Reply:
Thank you. Please help me resolve this thread.